mail/dovecot2/Portfile

```diff
@@ -67,7 +90,9 @@ pre-configure {
     # system -W ${worksrcpath} "gettextize -f"
     xinstall -W ${worksrcpath} ${prefix}/share/gettext/config.rpath .
 }
-
+if {${check.os.major} > ${os.major}} {
```

Comment:
I would incorporate this setting into the block above that tests the system and adds the pandoc dependency. It's just clearer to see it all in one place that way.

Reply:
I had my doubts at to whether I should split it into two or keep it in one place. In the end I split it up because the changes now are where they are logically used

Reply:
Yeah, that won't fly, usually. I would consider making pandoc included by default on all systems, and then remove the dep and disable the use of it in one block. Something like this pseudocode:
```
if {wrong system test} {
    delete build dep on pandoc
    disable build of port with pandoc
}
```
(Don't use that code, it's pseudocode for logically following what I'm talking about.

Reply:
there's a suggestion to remove pandoc completely : 
     https://trac.macports.org/ticket/58890
so the ? is : a hack to remove it on LibcxxOnOlderSystems -or- total removal

Reply:
```
set check.os.major  13
if {${os.platform} eq "darwin" && ${check.os.major} <= ${os.major}} {
    depends_build-append \
                    port:pandoc
} else {
    configure.env   PANDOC=false
}
```
```
if {${os.platform} eq "darwin" && ${check.os.major} > ${os.major}} {
    depends_build-delete \
                    port:pandoc
    configure.env   PANDOC=false
}
```

Reply:
> I would consider making pandoc included by default on all systems, and then remove the dep and disable the use of it in one block. 

The pandoc "dependency" is to support a maintainer-only function (building 2 doc files that are included in the release distributions) and it has an obnoxiously large dependency (Haskell's "stack") that is x86_64-only and appears likely to routinely break on older OS versions. 

I have tested building dovecot2 from source via MacPorts with pandoc disabled on High Sierra producing exactly the same files as building with pandoc enabled.

Reply:
so : remove pandoc completely is the best course of action ?!?

Reply:
Yes. That's why I asked for that and provided a patch to do so  in https://trac.macports.org/ticket/58890.

Reply:
Ports have maintainers exactly to resolve decisions like that one. Either approach is fine.

Reply:
well, the whole dovecot2 system is currently nomaintainer, so I guess we'll have to make all of the decisions in this forum.